docs/developer-guide/debugging-remote-environment.md

```diff
@@ -21,7 +21,7 @@ curl -sSfL https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/i
 Connect to one of the services, for example, to debug the main ArgoCD server run:
 ```shell
 kubectl config set-context --current --namespace argocd
-telepresence helm install # Installs telepresence into your cluster
+telepresence helm install --set agent.securityContext={} # Installs telepresence into your cluster
```

Comment:
Can someone confirm this to be working? I'm getting an error since `{}` ends up being converted to `["{}"]`, which causes traffic-manager to crashloop since it's not a valid security context. The only workaround I found is to pipe in a template: `printf "agent:\n  securityContext: {}\n" | telepresence helm install -f -`